src/commands/sources/query.ts

```diff
@@ -36,12 +36,14 @@ export class QuerySourcesCommand extends Command {
     }),
   };
 
-  protected render(results: QueryResult[]): void {
+  protected render(results: QueryResult[], catalog: Catalog): void {
     const rowsPerDistribution = results.map((result: QueryResult): Row[] => {
       return result.terms.map(
         (term: Term): Row => {
           return {
-            distributionTitle: result.dataset.distribution.url.toString(),
+            datasetTitle:
+              catalog.getDatasetByDistributionIri(result.distribution.iri)
+                ?.name ?? '',
```

Comment:
Super ugly, but this is what `gts fix` makes of it.

Reply:
😢 😄